.circleci/config.yml

```diff
@@ -41,9 +41,10 @@ executors:
     docker:
       - image: okteto/golang-ci:2.7.0
     environment:
-      OKTETO_CONTEXT: https://staging.okteto.dev
-      OKTETO_USER: cindylopez
-      OKTETO_APPS_SUBDOMAIN: staging.okteto.dev
+      OKTETO_CONTEXT: https://okteto.staging.dev.okteto.net
+      OKTETO_USER: staging-okteto-bot
```

Comment:
The user would be `okteto-bot`, we are not prefixing the okteto bot user with the release name.

Also, in staging we had configure `cindylopez` to allow 300 namespaces. Would it be needed for this user too?

Reply:
The user is `staging-okteto-bot`, I have been testing it. It could be because of the `namespacePrefix` properly.
Since `okteto-bot` doesn't have a limit for namespaces, we don't need to configure extra quotas

Reply:
Oh, that makes sense, let me try to remove it!

Reply:
Looking good, I love deleting code :-) 
Thanks for the tip!